src/util/ChartUtils.ts

```diff
@@ -100,9 +100,8 @@ export const calculateActiveTickIndex = (
   let index = -1;
   const len = ticks?.length ?? 0;
 
-  // if there are 1 or less ticks ticks then the active tick is at index 0
-  if (len <= 1) {
-    return 0;
+  if (len === 0) {
+    return index;
```

Comment:
this will return `-1` now yeah? How is that better than 0?

Reply:
I removed what seemed to be unnecessary code. 
I pushed the code again because there seemed to be an issue with the previous modifications